odl/util/testutils.py

```diff
@@ -229,6 +229,11 @@ def is_subdict(subdict, dictionary):
         reason='Need --benchmark option to run'
     )
 
+    skip_if_no_contrib = pytest.mark.skipif(
```

Comment:
Yeah, I think the same. This was just a WIP push while googling for a better solution.